lib/chef/resource/registry_key.rb

```diff
@@ -150,6 +151,7 @@ class RegistryKey < Chef::Resource
       }
       property :recursive, [TrueClass, FalseClass], default: false
       property :architecture, Symbol, default: :machine, equal_to: %i{machine x86_64 i386}
+      property :only_record_changes, [TrueClass, FalseClass], default: false
```

Comment:
This feels like it needs more documentation.

Reply:
yes please. I thought there was a `:help` option to properties that was used for docs...